impl/e2e-tests/clear-site-state.json

```diff
@@ -0,0 +1,51 @@
+{
+  "events": [
+    {
+      "seconds": 1,
+      "site": "a.example",
+      "event": "saveImpression",
+      "options": { "histogramIndex": 0 }
+    },
+    {
+      "seconds": 2,
+      "site": "advertiser-1.example",
+      "event": "measureConversion",
+      "options": {
+        "aggregationService": "https://agg-service.example",
+        "epsilon": 0.1,
+        "histogramSize": 1
+      },
+      "expected": [1]
+    },
+    {
+      "seconds": 3,
+      "event": "clearBrowsingHistoryForAttribution",
+      "sites": ["advertiser-1.example"],
+      "forgetVisits": false
+    },
+    {
+      "seconds": 4,
+      "site": "advertiser-1.example",
+      "event": "measureConversion",
+      "$comment": "re-run query, expecting budget to have been zeroed",
+      "options": {
+        "aggregationService": "https://agg-service.example",
+        "epsilon": 0.1,
+        "histogramSize": 1
+      },
+      "expected": [0]
```

Comment:
The reason that this fails is that the site lost its budget.

Would it make sense to have another test where an impression is removed instead?  You would have to advance time by an epoch between the time of the clear and the `measureConversion()` call, but I think that this is doable.